### PR TITLE
Fix some Pylint Warnings

### PR DESCRIPTION
**Pull request by hf-kklein**
This PR fixes some pylint warnings.
It does not change the behaviour in any way.
See the single commit messages for details.

### CODE REVIEW

**Comment by luiscoms** [comment]
Thanks for contributing @hf-kklein 
I will check why the tests are not running

File: healthcheck/environmentdump.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from .security import safe_dict
 
 
-class EnvironmentDump(object):
```
Comment (hf-kklein) [inline healthcheck/environmentdump.py:13]: maybe this breaks python2

**Comment by luiscoms** [comment]
Now you can rebase your branch in order to ensure tests are passing

File: healthcheck/healthcheck.py [hunk 2]
```diff
@@ -111,12 +113,10 @@ def run(self, check=None):
                 message = self.success_handler(results, **custom_section)
 
             return message, self.success_status, self.success_headers
-        else:
-            message = 'NOT OK'
-            if self.failed_handler:
-                message = self.failed_handler(results, **custom_section)
-
-            return message, self.failed_status, self.failed_headers
+        message = "NOT OK"
```
Comment (hf-kklein) [inline healthcheck/healthcheck.py]:
```suggestion
        message = 'NOT OK'
```

File: healthcheck/security.py [hunk 3]
```diff
@@ -18,8 +18,8 @@ def safe_dict(dictionary, blacklist=('key', 'token', 'pass'), max_deep=5):
     for key in dictionary.keys():
         if isinstance(dictionary[key], Mapping):
             result[key] = safe_dict(dictionary[key], blacklist, max_deep - 1)
-        elif any([b in key.lower() for b in blacklist]):
-            result[key] = '********'
+        elif any(b in key.lower() for b in blacklist):
+            result[key] = "********"
```
Comment (hf-kklein) [inline healthcheck/security.py]:
```suggestion
            result[key] = '********'
```

File: healthcheck/tornado_handler.py [hunk 4]
```diff
@@ -11,5 +11,6 @@ def initialize(self, checker):
     def get(self, *args, **kwargs):
         message, status_code, headers = self.checker.run(*args, **kwargs)
         self.set_status(status_code)
-        [self.set_header(k, v) for k, v in headers.items()]
+        for k, v in headers.items:
```
Comment (hf-kklein) [inline healthcheck/tornado_handler.py]:
```suggestion
        for k, v in headers.items():
```

**Comment by hf-kklein** [comment]
> Now you can rebase your branch in order to ensure tests are passing

@luiscoms done :) it was one actual error and one violation of a quoting convention. it's all green now ✔

File: healthcheck/healthcheck.py [hunk 5]
```diff
@@ -126,9 +126,9 @@ def run_check(self, checker):
                 passed, output = timeout(self.error_timeout, 'Timeout error!')(checker)()
             else:
                 passed, output = checker()
-        except Exception as e:
-            logger.exception(e)
-            passed, output = self.exception_handler(checker, e)
+        except Exception as exc:
+            logger.exception(exc)
```
Comment (luiscoms) [inline healthcheck/healthcheck.py]: This log could be error, not exception

File: healthcheck/healthcheck.py [hunk 6]
```diff
@@ -28,7 +28,7 @@ def json_success_handler(results, *args, **kw):
         'timestamp': time.time(),
         'results': results,
     }
-    [data.update({k: v}) for k, v in kw.items()]
+    data.update(kw)
```
Comment (luiscoms) [inline healthcheck/healthcheck.py:31]: I am not sure that this function is covered by tests, so this update should be tested

**Review by luiscoms — CHANGES_REQUESTED** [review]

File: healthcheck/healthcheck.py (hunk 5 above)
Comment (hf-kklein) [reply healthcheck/healthcheck.py]: 432e8f347b45e34fc732a9a6f00092a90e4a43e0

File: healthcheck/healthcheck.py (hunk 6 above)
Comment (hf-kklein) [reply healthcheck/healthcheck.py:31]: 9b811386193c9a147b6420102bf5040f0b877e88

**Review by luiscoms — APPROVED** [review]

**Pull request merged**